include/atomics/sender_cadmium.hpp

```diff
@@ -0,0 +1,183 @@
+/**
+ * Cristina Ruiz Martin
+ * ARSLab - Carleton University
+ */
+
+#ifndef __SENDER_CADMIUM_HPP__
+#define __SENDER_CADMIUM_HPP__
+
+#include <cadmium/modeling/ports.hpp>
+#include <cadmium/modeling/message_bag.hpp>
+#include <limits>
+#include <math.h> 
+#include <assert.h>
+#include <memory>
+#include <iomanip>
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <chrono>
+#include <algorithm>
+#include <limits>
+#include <random>
+
+#include "../data_structures/message.hpp"
+
+using namespace cadmium;
+using namespace std;
+
+//Port definition
+struct Sender_defs {
+	struct packetSentOut: public out_port<Message_t> {
+	};
+	struct ackReceivedOut: public out_port<Message_t> {
+	};
+	struct dataOut: public out_port<Message_t> {
+	};
+	struct controlIn: public in_port<Message_t> {
+	};
+	struct ackIn: public in_port<Message_t> {
+	};
+};
+
+template<typename TIME>
+class Sender {
```

Comment:
check variable, please. All should be small letter

Reply:
we need not change Class name